### PR TITLE
start of the community center

### PR DESCRIPTION
**Pull request by ItzKatze**
- added Bits-Shop
- added Booster Cookie Shop
- (small) added layout of the bestiary page

### CODE REVIEW

File: type-village/src/main/java/net/swofty/type/village/gui/elizabeth/GUIBitsShop.java [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+                        lore.add(" ");
+                        lore.add("§eClick to trade!");
+                        return ItemStackCreator.updateLore(itemStack, lore);
+                    }
+                });
+                index2++;
+            }
+        }
+        SubCategorys[] allSubCategorys = SubCategorys.values();
+        int index3 = 0;
```
Comment (Swofty-Developments) [inline type-village/src/main/java/net/swofty/type/village/gui/elizabeth/GUIBitsShop.java:210]: fix variable names

File: type-village/src/main/java/net/swofty/type/village/gui/elizabeth/GUIBitsShop.java [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+                        ArrayList<String> lore = new ArrayList<>(itemStack.build().getLore().stream().map(StringUtility::getTextFromComponent).toList());
+                        lore.add(" ");
+                        lore.add("§7Cost");
+                        lore.add("§b" + StringUtility.commaify(bitItems.price) + " Bits");
+                        lore.add(" ");
+                        lore.add("§eClick to trade!");
+                        return ItemStackCreator.updateLore(itemStack, lore);
+                    }
+                });
+                index2++;
```
Comment (Swofty-Developments) [inline type-village/src/main/java/net/swofty/type/village/gui/elizabeth/GUIBitsShop.java:206]: fix variable names

File: type-village/src/main/java/net/swofty/type/village/gui/elizabeth/GUIBitsShop.java [hunk 3]
```diff
@@ -0,0 +1,304 @@
...
+                    if (slot != 13) {
+                        return ItemStackCreator.getStack("§8▲ §7Categories", Material.GRAY_STAINED_GLASS_PANE, 1, "§8▼ §7Items");
+                    } else {
+                        return ItemStackCreator.getStack("§8▲ §7Categories", Material.GREEN_STAINED_GLASS_PANE, 1, "§8▼ §7Items");
+                    }
+                }
+            });
+        }
+        BitItems[] allBitItems = BitItems.values();
+        int index2 = 0;
```
Comment (Swofty-Developments) [inline type-village/src/main/java/net/swofty/type/village/gui/elizabeth/GUIBitsShop.java:172]: fix variable names

File: type-village/src/main/java/net/swofty/type/village/gui/elizabeth/GUIBitsShop.java [hunk 4]
```diff
@@ -0,0 +1,304 @@
...
+                    ItemStack.Builder itemstack = subCategorys.item;
+                    ArrayList<String> lore = new ArrayList<>(itemstack.build().getLore().stream().map(StringUtility::getTextFromComponent).toList());
+                    if (!Objects.equals(lore.getLast(), "§eClick to browse!")) {
+                        lore.add(" ");
+                        lore.add("§eClick to browse!");
+                    }
+                    return ItemStackCreator.updateLore(itemstack, lore);
+                }
+            });
+            index3++;
```
Comment (Swofty-Developments) [inline type-village/src/main/java/net/swofty/type/village/gui/elizabeth/GUIBitsShop.java:230]: fix variable names

**Review by Swofty-Developments — CHANGES_REQUESTED** [review]

File: type-village/src/main/java/net/swofty/type/village/gui/elizabeth/GUIBitsShop.java (hunk 1 above)
Comment (ItzKatze) [reply type-village/src/main/java/net/swofty/type/village/gui/elizabeth/GUIBitsShop.java]: done

**Pull request merged**